docs/adding-ads.md

```diff
@@ -0,0 +1,23 @@
+# Adding Super Cool Ads for Display in Posts
+
+After you [configure the plugin](configuration.md) to set the number of paragraphs before each ad insertion, and the number of available ad insertions, visit **Appearance** > **Widgets** to create the ads themselves. You'll see a number of Inserted Ad Positions as Widget areas where you can place widgets. You can set the number of available Inserted Ad Positions in the [plugin options panel](configuration.md). Each Inserted Ad Position widget area can have one and only one ad.
+
+To create an ad, begin by adding a Text Widget to one of the numbered Inserted Ad Positions:
```

Comment:
This area should note that for demonstration purposes, we're using a text widget, but other widget types are supported. Examples can be an ad widget from [DoubleClick for WordPress](https://wordpress.org/plugins/doubleclick-for-wp/), a newsletter signup widget, an image widget, a related posts widget, or anything else.